node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.0.3-preview",
+  "version": "3.0.4-preview.0",
```

Comment:
I think this should just be 3.0.4-preview

Reply:
This was implemented to resolve the issue with task-lib in common packages without using _peerDependencies_.
When we use this pattern _"3.0.3-preview"_ for versioning we are not able to use ^ in the dependencies to allow installing a newer version.
If we implement it this way _"3.0.4-preview.0"_ we would be able to use semantic versioning syntax and resolve the issue with task-lib version conflicts.

Reply:
Ah, I missed the email explaining this - my mistake!